odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW

```diff
@@ -1167,16 +1348,53 @@ PROCEDURE FMAUTHFEEFIELDS
   SDAUTHFEEUPDATE(TMPLOOP)=FMSUCCESSFUL
 END [PROCEDURE]
 
+PROCEDURE FMODTAMOUNT
+[* Update OVERDRAFTTOLERANCE amount in the SHARE record
+*]
+ FMERROR=""
+ FMUPDATEERRORODTAMOUNT=FALSE
+
+ IF SDODTSTATUS(TMPLOOP)<>SDODTNEWSTATUS(TMPLOOP) THEN
+  DO
+   IF SDODTNEWSTATUS(TMPLOOP)=FALSE THEN
+    NEWODTAMOUNT=$0.00
+   ELSE
+    DO
+     NEWODTAMOUNT=SDOVERDRAFTTOLERANCE(TMPLOOP)
+     PARAMINDEX=1
+     PARAMFOUND=FALSE
+     WHILE (PARAMFOUND=FALSE AND PARAMINDEX<=PARAMOTACOUNT)
+      DO
+       IF PARAMOTASHARETYPE(PARAMINDEX)=SDTYPE(TMPLOOP) THEN
+        DO
+         PARAMFOUND=TRUE
+         NEWODTAMOUNT=PARAMOTAAMOUNT(PARAMINDEX)
+        END
+       PARAMINDEX=PARAMINDEX+1
+      END
+      
+    END
+
+   FMPERFORM REVISE SHARE SDID(TMPLOOP) (0,0,FMERROR)
```

Comment:
Should the FMPERFORM be wrapped in a PARAMFOUND=TRUE clause?

Reply:
I don't believe so.  As written and designed, it may update the Overdraw Tolerance amount to $0.00 when opting-out.  Also, if the param is not found, we will just update the field with the current value -- so no actual update.  Sound good?

Reply:
Yep